backend/handlers/handlers.go

```diff
@@ -239,10 +239,10 @@ func NewHandlers(
 	getAPIRouterNoError(apiRouter)("/electrum/check", handlers.postElectrumCheck).Methods("POST")
 	getAPIRouterNoError(apiRouter)("/socksproxy/check", handlers.postSocksProxyCheck).Methods("POST")
 	getAPIRouterNoError(apiRouter)("/exchange/by-region/{code}", handlers.getExchangesByRegion).Methods("GET")
-	getAPIRouterNoError(apiRouter)("/exchange/deals", handlers.getExchangeDeals).Methods("GET")
-	getAPIRouterNoError(apiRouter)("/exchange/buy-supported/{code}", handlers.getExchangeBuySupported).Methods("GET")
+	getAPIRouterNoError(apiRouter)("/exchange/deals/{action}/{code}", handlers.getExchangeDeals).Methods("GET")
+	getAPIRouterNoError(apiRouter)("/exchange/supported/{code}", handlers.getExchangeSupported).Methods("GET")
```

Comment:
if code is an account code, why are these functions not living in btc/handlers.go? :nerd_face:

Reply:
On second thought, there is a reason: both these functions need `backend.HTTPClient()`. In general, all the exchange-related functions that use the account code also need some backend element. The only exception is `getExchangeSupported()` which could be easily moved, but I'm not sure if it makes much sense to keep all the exchange-related handlers in one place, except this. wdyt?